app/views/subscriptions/_channel_switch.html.haml

```diff
@@ -0,0 +1,18 @@
+- # locals:
+- channel || raise('no channel given.')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.